dateparser/custom_language_detection/fasttext.py

```diff
@@ -0,0 +1,31 @@
+import fasttext
+import os
+
+from dateparser_cli.fasttext_manager import fasttext_downloader
+from dateparser_cli.utils import date_parser_model_home
+
+
+_supported_models = ["large.bin", "small.bin"]
+_downloaded_models = os.listdir(date_parser_model_home)
+
+_model_path = None
+
+for downloaded_model in _downloaded_models:
+    if downloaded_model in _supported_models:
+        _model_path = date_parser_model_home + "/" + downloaded_model
+
+if not _model_path:
+    fasttext_downloader(["small"])
```

Comment:
I think this needs to be updated as the argument is not a list. Also I think it's better to not download the model on module import - but only when we need to load it.